sdk/options.go

```diff
@@ -61,12 +61,18 @@ func (o *Options) init() error {
 		o.ConsoleLogger = l
 	}
 
-	bs, err := ioutil.ReadAll(o.Config)
-	if err != nil {
-		return err
+	var bs = []byte("{}")
```

Comment:
[nit] This should be the same as
```suggestion
	bs := []byte("{}")
```

where I think the `bs :=` variant is a bit more consistent with the overall code style

Reply:
Made the recommended adjustments and added a test case.